ddtrace/internal/writer/writer.py

```diff
@@ -321,7 +321,6 @@ def _send_payload(self, payload, count, client):
 
     def write(self, spans=None):
         # Queues an app-started event before the first ci-visibility/llmobs/trace payload is sent
-        telemetry_writer.app_started()
```

Comment:
We should also remove ` telemetry_writer.app_started()` from the profiler

Reply:
done